ci/conmon-scan-ecr-container.yml

```
@@ -0,0 +1,17 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: 18fgsa/concourse-task
+
+inputs: grype-scan-ignore-config
+- name: scan-source
+- name: image-source
```

```suggestion
inputs: 
- name: grype-scan-ignore-config
- name: scan-source
- name: image-source
```

This is how you add `grype-scan-ignore-config` as a new list item to the `inputs` here in YAML